src/ralph/accounts/admin.py

```diff
@@ -91,6 +92,21 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class StaffStatusFilter(BooleanFilter):
```

Comment:
Do we need to declare dozens of classes every time we want to add new filters? Maybe it would be better to create simple metaclass which will take away trivial params cases like (title, param_name) ?

For example:
```
filters.add(
    ('Active', 'is_active'), 
    ('Superuser', 'is_superuser'), 

etc
```